examples/tts/tacotron2.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import hydra
+import pytorch_lightning as pl
+
+from nemo.collections.tts.helpers.helpers import LogEpochTimeCallback
+from nemo.collections.tts.models import Tacotron2PTL
+from nemo.utils.exp_manager import exp_manager
+
+
+@hydra.main(config_path="conf", config_name="tacotron2")
+def main(cfg):
+    trainer = pl.Trainer(**cfg.pl.trainer)
+    exp_manager(trainer, cfg.get("exp_manager", None))
+    model = Tacotron2PTL(cfg=cfg.model, trainer=trainer)
+    lr_logger = pl.callbacks.LearningRateLogger()
+    epoch_time_logger = LogEpochTimeCallback()
```

Comment:
Should it be part of the Model? These callbacks are not necessary should someone just want to use our WaveGlow model. For that reason, I think it makes sense to separate things like callbacks and logger from a Model

Reply:
LR logger and Epoch time seems like general enough to be included with every model

Reply:
I have actually removed LR logger from waveglow because it uses a constant LR (without a scheduler) and LR Logger does not work without a scheduler. So I'm not sure how general they are.

Reply:
Hmm alright, lets keep LR schedule logger seperate for now, what about EpochTimeCallback()?